form.yml.erb

```diff
@@ -185,6 +185,7 @@ attributes:
     widget: "select"
     label: "JupyterLab Version"
     options:
+      - [ "3.0",  "app_jupyter/3.0.16" ]
```

Comment:
Even if it's default, we still want to enable the option for 3.0 right?

Reply:
Yes, I will create another branch for that.

Reply:
:+1: